plugins/azure/azure.plugin.zsh

```diff
@@ -0,0 +1,67 @@
+# AZ Get Subscritions
+function azgs() {
+  az account show --output tsv --query 'name'
+}
+
+# AZ Subscription Selection
+alias azss="az account set --subscription"
+
+
+function az_subscriptions() {
+  [[ -f "${AZURE_CONFIG_DIR:-$HOME/.azure}/accessTokens.json" ]] || return 1
+  az account list  --all --output tsv --query '[*].name'
+}
+
+function _az_subscriptions() {
+  reply=($(az_subscriptions))
+}
+compctl -K _az_subscriptions azss
+
+# Az prompt
+function prompt_azure() {
```

Comment:
Convention seems to be `<plugin>_prompt_info()`
```suggestion
function azure_prompt_info() {
```

And would probably need to add the stub at `lib/prompt_info_functions.zsh`

Reply:
Yeah unless I'm missing something, I don't see a canonical way to update a prompt, I"m using POWERLEVEL9K (damn deprecated too, needs 10k)

Reply:
oh wow, yeah it seems to be a theme decision, the powerlevel10k integration is pretty nice (loads the current subscriptions in the right prompt only if I type `az`)

I'll keep `azgs` as variable to use regardless in a RPROMPT, but that simplifies a lot.